src/main/main.ts

```diff
@@ -84,7 +84,11 @@ const createWindow = async () => {
     },
   });
 
-  initialiseIpcHandlers(mainWindow);
+  const ipcContext: IpcContext = {
```

Comment:
Does `IpcContext` need to be imported here?

Reply:
Yup, done thanks